c2corg_ui/templates/article/edit.html

```diff
@@ -80,17 +80,17 @@ <h2 class="heading show-phone" translate>general informations</h2>
                 <label><span translate>article_type</span> *</label>
                 % if updating_doc:
                   <select ng-if="userCtrl.auth.isModerator()" class="form-control" ng-model="article.article_type"
-                          ng-options="type as mainCtrl.translate(type) for type in ${article_types}" ><option value=""></option></select>
+                          ng-options="type as mainCtrl.translate(type) for type in ${article_types} | translate" ><option value=""></option></select>
```

Comment:
I don't think this correction (and the other similar ones below) is correct because the options are already translated, see ``type as mainCtrl.translate(type)`` => it does not make sense to make 2 calls to the ``translate`` directive.

I don't know exactly why the translations are not done as expected (are they?). For instance in other editing forms they seem to work as is:
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/waypoint/edit.html#L296

We already had this problem for some other attributes in the editing forms. I think the problem is related to the direct use of the mako variable ``${article_types}`` in ``ng-options``.
An alternate syntax that seems to work is to define an angular variable using ``ng-init`` and to use  it in the ``ng-options``. See for instance 
https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/outing/edit.html#L269-L272